From 94ab405ecf38fb47edcb08afb523e8c1fc7fb44e Mon Sep 17 00:00:00 2001
From: Oliver Steffen <osteffen@redhat.com>
Date: Wed, 14 Aug 2024 09:53:49 +0200
Subject: [PATCH] NetworkPkg/DxeNetLib: Reword PseudoRandom error logging

RH-Author: Oliver Steffen <osteffen@redhat.com>
RH-MergeRequest: 67: NetworkPkg/DxeNetLib: adjust PseudoRandom error logging
RH-Jira: RHEL-45899
RH-Commit: [2/2] 0d465ca0ea00598e6826446cd08e890c2ae4bea7 (osteffen/edk2)

The word "Failed" is used when logging tired Rng algorithms.
These mostly non-critical messages confused some users.

Reword it and also add a message confirming eventual success to
deescalate the importance somewhat.

Signed-off-by: Oliver Steffen <osteffen@redhat.com>

patch_name: edk2-NetworkPkg-DxeNetLib-Reword-PseudoRandom-error-loggi.patch
present_in_specfile: true
location_in_specfile: 41
---
 NetworkPkg/Library/DxeNetLib/DxeNetLib.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/NetworkPkg/Library/DxeNetLib/DxeNetLib.c b/NetworkPkg/Library/DxeNetLib/DxeNetLib.c
index 9acc21caeb..73ddf8b0fb 100644
--- a/NetworkPkg/Library/DxeNetLib/DxeNetLib.c
+++ b/NetworkPkg/Library/DxeNetLib/DxeNetLib.c
@@ -952,12 +952,13 @@ PseudoRandom (
         //
         // Secure Algorithm was supported on this platform
         //
+        DEBUG ((DEBUG_VERBOSE, "Generated random data using secure algorithm %d: %r\n", AlgorithmIndex, Status));
         return EFI_SUCCESS;
       } else if (Status == EFI_UNSUPPORTED) {
         //
         // Secure Algorithm was not supported on this platform
         //
-        DEBUG ((DEBUG_VERBOSE, "Failed to generate random data using secure algorithm %d: %r\n", AlgorithmIndex, Status));
+        DEBUG ((DEBUG_VERBOSE, "Unable to generate random data using secure algorithm %d not available: %r\n", AlgorithmIndex, Status));
 
         //
         // Try the next secure algorithm
